src/main/resources/templates/azure-cosmosdb/CosmosdbController.java

```diff
@@ -10,11 +10,14 @@
 @RequestMapping("/cosmosdb")
 public class CosmosdbController {
 
+    private static final User USER = new User("id", "first-name", "last-name", "address");
+
     @Autowired
     private UserRepository repository;
 
     @GetMapping(value = "/users")
     public List<User> getUsers() {
+        this.repository.save(USER);
```

Comment:
Save data for every get request? Move initialize to constructor, and inject repository by constructor

Reply:
Roger that.